python/MOCK_sensors.py

```diff
@@ -122,8 +131,151 @@ def publish(self):
 
         if rospy.get_param('sensor_noise', default=False):
             data = self.add_noise(data)
+
+        brokenWindSensorStatus = rospy.get_param('broken_wind_sensors', default="not_broken")
```

Comment:
Would be best if you put this into another function to make the `publish` function more clear.

```
brokenWindSensorStatus = rospy.get_param('broken_wind_sensors', default="not_broken")
if brokenWindSensorStatus != "not_broken":
    # update
```